src/algorithms/digi/SiliconTrackerDigi.cc

```diff
@@ -66,31 +66,43 @@ void SiliconTrackerDigi::process(
             continue;
         }
 
-        if (cell_hit_map.count(sim_hit.getCellID()) == 0) {
-            // This cell doesn't have hits
-            cell_hit_map[sim_hit.getCellID()] = {
+        bool bucket_found = false;
+        if (cell_hit_map.count(sim_hit.getCellID()) == 1) {
```

Comment:
This should have been
```с++
        if (cell_hit_map.count(sim_hit.getCellID()) >= 1) {
```
just drop this condition and keep the inner loop.

Reply:
Actually, protecting against an implicit insertion in `operator[]` is kind of a good manner, even if it doesn’t matter yet. This is fine to keep, to stay explicit.

Reply:
I actually didn't think about it much, I just kept (well, inverted) the pre-existing `if (cell_hit_map.count(sim_hit.getCellID()) == 0)` conditional